README.md

```diff
@@ -1,245 +1,245 @@
+one this repository to your desired folder https://github.com/techmoves/hello-microvese.git
+
+Run the command npm install to install necessary dependencies contained in the package.json file
+
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+Install this project with these cmd:
+
+     1.cd Desktop/
+     2git clone using repository link
+     3.cd Hello-world/
+     4.git branch branch_name
+     5.git checkout branch_name
+
+gitbash and clone the repo using HTTP link
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+<!-- ### Usage
+
+To run the project, execute the following command: -->
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+     1.Run tests
+     2.To run tests run :
+     3.npx hint .
+     4.npx stylelint "*/.{css,scss}"
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Phelelani**
+
+- GitHub: [@githubhandle](https://github.com/techmoves)
+- Twitter: [@twitterhandle](https://twitter.com/Dopezonke_)
+- LinkedIn: [LinkedIn][(https://linkedin.com/in/phelelani-ndlovu-883916160)]
+
+
+
+
+
+<!-- SUPPORT -->
+
+
+
+> Write a message to encourage readers to support your project
+> to new student    from microverse they veiw this project for review
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank... microverse and the whole team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

```

Comment:
As requested by the previous reviewers please 🙏 remove `Note:` from the LICENCE section of the README. Note 🗒️ is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it 👍. This is from this [PR](https://github.com/techmoves/hello-microverse/pull/3)